tools/donate-cpu.py

```diff
@@ -289,7 +289,20 @@ def scanPackage(workPath, cppcheckPath, jobs):
                     stacktrace = stdout[gdb_pos:]
                 else:
                     stacktrace = stdout[last_check_pos:]
-        return -1, stacktrace, '', -1, options
+        return -11, stacktrace, '', -11, options
+    if returncode != 0:
+        print('Error!')
+        return returncode, '', '', returncode, options
+    if stderr.find('Internal error: Child process crashed with signal ') > 0:
```

Comment:
Those greps require an English language environment?

Reply:
It#s the same string, that was already used in the crash detection and is written exactly like this by cppcheck, so there should be locale issues.